sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -687,13 +687,48 @@ def get_cmis_media_lanes_mask(self, api, appl, lport, subport):
 
         return media_lanes_mask
 
-    def is_appl_reconfigure_required(self, api, app_new):
+    def set_dp_decommision_pending(self, lport, decomm_pending):
```

Comment:
@AnoopKamath please do what the function name indicates. create more func if rquired.